src/Views/BucketPanel/EntryDetail.tsx

```diff
@@ -181,6 +285,153 @@ export default function EntryDetail(props: Readonly<Props>) {
           onRemoved={() => history.push(`/buckets/${bucketName}`)}
         />
       )}
+
+      <Modal
```

Comment:
Let's move the form it's logic into the `UploadFileForm.ts` and `UploadFileForm.css`. See src/Components/Entry.

Reply:
@hussnainzafar the idea was to build it as a component that encapsulate the logic. You create it as a html form with properties. Please, move all your states to  `UploadFileForm.ts`  and handle the upload from  there. You havent' created `UploadFileForm.css` and keeps using styles in UploadFileForm.ts.